recordset/recordset.js

```diff
@@ -112,8 +113,15 @@
                 $window.scrollTo(0, 0);
 
                 recordsetModel.page = page;
-                recordsetModel.rowValues = page.tuples.map(function (tuple, index, array) {
-                    return tuple.values;
+                recordsetModel.rowValues = page.tuples.map(function(tuple, index, array) {
+                    var row = [];
+                    tuple.values.forEach(function(value, index) {
+                        row.push({
+                            isHTML: tuple.isHTML[index],
+                            value: (tuple.isHTML[index]? $sce.trustAsHtml(value) : value)
+                        });
+                    });
+                    return row;
```

Comment:
Why are we combing the `value` and `isHTML` into an object ? If thats the case then `tuple.values` should return an array of objects by default that contains the `value` and `isHTML` instead of a plain string array. Or else we should be handling this in the angular HTML template itself.